core/src/test/java/org/elasticsearch/recovery/RecoveryWhileUnderLoadIT.java

```diff
@@ -255,19 +256,21 @@ public void testRecoverWhileRelocating() throws Exception {
             logger.info("--> refreshing the index");
             refreshAndAssert();
             logger.info("--> verifying indexed content");
-            iterateAssertCount(numShards, indexer.totalIndexedDocs(), 10);
+            iterateAssertCount(numShards, (int) indexer.totalIndexedDocs(), 10);
```

Comment:
left over - I tried to move away from the long param, but realized it caused more pain that gains. reverting...